client/app/src/main/java/com/example/criengine/Activities/RootActivity.java

```diff
@@ -51,21 +66,25 @@ protected void onCreate(Bundle savedInstanceState) {
         // If returning from another activity, this can control which screen to navigate to.
         if (getIntent().getExtras() != null) {
             int index = ((PAGE) getIntent().getSerializableExtra("Index")).getValue();
-            viewPager.setCurrentItem(index);
+            viewPager.setCurrentItem(index, false);
         } else {
             // Returns to my books. This will be the home screen.
-            viewPager.setCurrentItem(PAGE.MY_BOOKS.getValue());
+            viewPager.setCurrentItem(PAGE.MY_BOOKS.getValue(), false);

```

Comment:
`false` changes the current page without animation/scrolling.
It looks weird when this Activity opens up on page 0 and immediately scrolls to `index` or `MY_BOOKS`

Reply:
Ayy thats what we needed. ty